zcnbridge/authorizer.go

```diff
@@ -1,34 +0,0 @@
-package zcnbridge
-
-import (
-	"encoding/json"
-
-	"github.com/0chain/gosdk/zcnbridge/http"
-	"github.com/pkg/errors"
-)
-
-type AuthorizerNodeResponse struct {
-	ID  string `json:"id"`
-	URL string `json:"url"`
-}
-
-// GetAuthorizers Returns authorizers
-func GetAuthorizers() ([]*AuthorizerNodeResponse, error) {
-	var (
-		nodes []*AuthorizerNodeResponse
-	)
```

Comment:
Define the variable in the place where it is used.

Reply:
fixed